src/lesson10/exceprions/userValidationService/UserValidator.java

```diff
@@ -0,0 +1,41 @@
+package lesson10.exceprions.userValidationService;
+
+public class UserValidator {
+
+    private void assertUserNotNull(User user) {
+        if (user.getFirstName() == null
+                || user.getLastName() == null
+                || user == null) {
```

Comment:
Check for `user == null` first because otherwise you'll still get NullPointerException
Conditions are evaluated from left to right, i.e. the first check will be the `user.getFirstName() == null` and if the user is null you'll get an exeption

Reply:
I understood the importance of consistency, corrected it.